scripts/validate/check_version.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euo pipefail
+
+echo "Checking kubernetes version replacement / pinning"
+
+readonly kubeversion=$(go list -m -f '{{.Version}}' k8s.io/kubernetes)
+
+if [[ "${kubeversion}" =~ '^v.*' ]]; then
+	echo "Kube Version: ${kubeversion} in unexpected format, must start with v"
+	exit 1
+fi
+
+echo "Found kubernetes version: ${kubeversion}"
+
+readonly sha=$(curl http://api.github.com/repos/kubernetes/kubernetes/tags -L -s |jq -r --arg kubeversion "${kubeversion}" '.[] |select (.name==$kubeversion) | .commit.sha')
+
+if [[ ! "${sha}" =~ ^[0-9a-f]{40}$ ]]; then
+	echo "Kube Sha: ${sha} in unexpected format"
```

Comment:
Since  this PR merged,  we can not  run the test successfully. 

error msg is:

```
Kube Sha:  in unexpected format
```

Reply:
Can you run:

`bash -x scripts/validate/check_version.sh`?

Reply:
(and post the output)

Reply:
https://app.circleci.com/pipelines/github/virtual-kubelet/virtual-kubelet/45/workflows/b58fdfa4-eb87-4a81-9ed4-615579d43e58/jobs/5237/steps